Apps/WebClient/src/NewClientApp/src/views/LogoutCompleteView.vue

```diff
@@ -0,0 +1,25 @@
+﻿<script setup lang="ts">
+import { useRoute, useRouter } from "vue-router";
+import { useConfigStore } from "@/stores/config";
+
+const route = useRoute();
+const router = useRouter();
+const configStore = useConfigStore();
+
+setTimeout(() => {
+    if (route.path == "/logoutComplete") {
+        router.push({ path: "/" });
+    }
+}, Number(configStore.webConfig.timeouts.logoutRedirect));
+</script>
+
+<template>
+    <v-container class="pt-12 text-center h-100">
```

Comment:
I think we'll want to add standard margins around the main content in App.vue in the future, so the top padding here won't be needed then.  It also doesn't look like `h-100` is having an effect here.